src/pages/commisionalBundles/list/CommissionBundlesTable.tsx

```diff
@@ -1,198 +1,212 @@
+    const addError = useErrorDispatcher();
+    const selectedParty = useAppSelector(partiesSelectors.selectPartySelected);
+    const setLoading = useLoading(LOADING_TASK_COMMISSION_BUNDLE_LIST);
+    const columns: Array<GridColDef> = buildColumnDefs(t, orgInfo.types.isPsp, orgInfo.types.isEc);
+    const [listFiltered, setListFiltered] = useState<BundlesResource>(emptyCommissionBundleList);
+    const [page, setPage] = useState(0);
+    const brokerCode = selectedParty?.fiscalCode ?? '';
+    const [isFirstRender, setIsFirstRender] = useState<boolean>(true);
 
-  const setLoadingStatus = (status: boolean) => {
-    setLoading(status);
-  };
+    const setLoadingStatus = (status: boolean) => {
+        setLoading(status);
+    };
 
-  const pageLimit = 5;
-  const getBundleList = (newPage?: number) => {
-    setLoadingStatus(true);
-    if (isFirstRender) {
-      setIsFirstRender(false);
-    }
-    const mappedBundleType = mapBundle(bundleType);
-    // eslint-disable-next-line functional/no-let
-    let promise;
-    if (isPsp()) {
-      promise = getBundleListByPSP(
-        mappedBundleType,
-        pageLimit,
-        bundleNameFilter,
-        newPage ?? page,
-        brokerCode
-      );
-    } else if (isEc()) {
-      promise = getCisBundles(
-        mappedBundleType,
-        pageLimit,
-        bundleNameFilter,
-        newPage ?? page,
-        mappedBundleType === TypeEnum.GLOBAL ? undefined : brokerCode
-      );
-    } else {
-      promise = new Promise<BundlesResource>((resolve) => resolve({}));
-    }
-    promise
-      .then((res) => {
-        if (res?.bundles) {
-          const formattedBundles = res?.bundles?.map((el, ind) => ({
-            ...el,
-            id: `bundle-${ind}`,
-            touchpoint: el.touchpoint ?? 'ANY',
-            paymentType: el.paymentType ?? 'ANY',
-          }));
-          setListFiltered({ bundles: formattedBundles, pageInfo: res.pageInfo });
+    const pageLimit = 5;
+    const getBundleList = (newPage?: number) => {
+        setLoadingStatus(true);
+        if (isFirstRender) {
+            setIsFirstRender(false);
+        }
+        const mappedBundleType = mapBundle(bundleType);
+        // eslint-disable-next-line functional/no-let
+        let promise;
+        if (orgInfo.types.isPsp) {
+            promise = getBundleListByPSP(
+                mappedBundleType,
+                pageLimit,
+                bundleNameFilter,
+                newPage ?? page,
+                brokerCode
+            );
+        } else if (orgInfo.types.isEc) {
+            promise = getCisBundles(
+                mappedBundleType,
+                pageLimit,
+                bundleNameFilter,
+                newPage ?? page,
+                mappedBundleType === TypeEnum.GLOBAL ? undefined : brokerCode
+            );
         } else {
-          setListFiltered([]);
+            promise = new Promise<BundlesResource>((resolve) => resolve({}));
+            addError({
```

Comment:
Instead of adding a new addError, I would change the empty promise resolve (line 92) with a reject so it uses the already defined addError at the end of the promise